client/src/components/Announcement.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+
+class Announcement extends Component {
+    constructor(props) {
+        super(props);
+    this.state = {
```

Comment:
nitpick: try to maintain consistent spacing.
Might be a good extension to try out: https://marketplace.visualstudio.com/items?itemName=jawandarajbir.react-vscode-extension-pack